public/styles/style.css

```diff
@@ -44,6 +57,178 @@ body {
   display: none !important;
 }
 
+section {
+  padding: 1rem;
+}
+
+h1, h2, h3 {
+  color: var(--mine-shaft);
+}
+
+/* ======================================= */
+/* member section */
+/* ======================================= */
+section p {
+  color: var(--neutral-2);
+  margin-bottom: 0;
+}
+
+.member-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  align-items: center;
+  list-style-type: none;
+}
+
+.member-list .member-item {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-right: 1rem;
+}
+
+.member-list .member-item img {
+  border-radius: 50%;
+  width: 3rem;
+  height: 3rem;
+  background-color: var(--neutral-2);
+  padding: 1rem;
+
+}
+
+.option-row {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+.option-row .btn-blue {
+  display: inline-block;
+  background-color: var(--regent-blue);
+  margin: 1rem 0;
+  flex: 1;
+  outline: none;
+  border: 0.1rem solid var(--regent-blue);
+  padding: 0.5rem;
+  color: var(--lochmara);
+  border-radius: 0.25rem;
+  transition: 0.2s ease-in;
+}
+
+.option-row .btn-blue:hover {
+  border: 0.1rem solid var(--regent-blue-darker);
+  outline-offset: 1rem;
+}
+
+.option-row .btn-blue:nth-child(1) {
+  margin-right: 1rem;
+}
+
+.option-row .btn-blue:nth-child(2)::before {
+  content: url('./assets/icons/icon-user.svg');
+  margin-right: 0.25rem;
+  width: 24px;
+  height: 24px;
+}
+/* ======================================= */
+/* Dialog
+/* ======================================= */
+dialog {
+  position: fixed;
+  bottom: 0;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 99999;
```

Comment:
Probeer hoge z-index properties te vermijden. Gebruik sowieso zo weinig mogelijk z-index, maar volgensmij is een z-index van 2 of 3 hier ook al voldoende.